sharing/handler.go

```diff
@@ -406,24 +407,41 @@ func (handler *Handler) handleHostingService(ctx context.Context, w http.Respons
 		}
 	}()
 
-	// e.g. http://mydomain.com/folder2/index.html with root="bucket1/folder1"
-	rootPath := strings.SplitN(root, "/", 2) // e.g. rootPath=[bucket1, folder1]
+	// e.g. http://mydomain.com/folder2/index.html with "storj_root:bucket1/folder1/"
+	rootPath := strings.SplitN(root, "/", 2) // e.g. rootPath=[bucket1, folder1/]
```

Comment:
I think this path construction could be in some sort of helpers that figures out the exact paths. Currently it's not quite clear what's happening.

Reply:
sure that's a good idea

Reply:
done